src/main/java/com/github/rahulsinghai/logback/kafka/KafkaAppenderConfig.java

```diff
@@ -59,6 +61,16 @@ protected boolean checkPrerequisites() {
             deliveryStrategy = new AsynchronousDeliveryStrategy();
         }
 
+        // Set system properties for JAAS and krb5 conf files
+        if (errorFree && producerConfig.get(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG) != null
+            && producerConfig.get(CommonClientConfigs.SECURITY_PROTOCOL_CONFIG).toString().contains("SASL")
+            && clientJaasConfPath != null) {
+            System.setProperty("java.security.auth.login.config", clientJaasConfPath);
```

Comment:
I am absolutely not into kerberos, but is this the only and desired to set it up for the kafka producer? I'd rather like to avoid changing jvm global properties as side-effect of an appender initalisation.